catalogs/catalog-lakehouse-iceberg/src/main/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/IcebergConfig.java

```diff
@@ -69,6 +71,28 @@ public class IcebergConfig extends Config {
           .booleanConf()
           .createWithDefault(true);
 
+  public static final ConfigEntry<String> ICEBERG_METRICS_STORAGE =
+      new ConfigBuilder(IcebergMetricsManager.ICEBERG_METRICS_STORAGE)
+          .doc("The storage to store Iceberg metrics")
+          .version("0.4.0")
+          .stringConf()
+          .createWithDefault(MemoryMetricsStorage.ICEBERG_METRICS_STORAGE_MEMORY_NAME);
+
+  public static final ConfigEntry<Integer> ICEBERG_METRICS_STORAGE_RETAIN_DAYS =
+      new ConfigBuilder(IcebergMetricsManager.ICEBERG_METRICS_STORAGE_RETAIN_DAYS)
+          .doc(
+              "The retain days of Iceberg metrics, the value not greater than 0 means retain forever")
+          .version("0.4.0")
+          .intConf()
+          .createWithDefault(-1);
+
+  public static final ConfigEntry<Integer> ICEBERG_METRICS_QUEUE_CAPACITY =
+      new ConfigBuilder(IcebergMetricsManager.ICEBERG_METRICS_QUEUE_CAPACITY)
+          .doc("The capacity for Iceberg metrics queues, should greater than 0")
+          .version("0.4.0")
+          .intConf()
```

Comment:
Could you use `checkValue` to guarantee the number is greater than 0?

Reply:
ok

Reply:
done